src/test/ui/nll/closure-requirements/propagate-approximated-shorter-to-static-comparing-against-free.stderr

```diff
@@ -72,14 +72,15 @@ LL | | }
 error[E0597]: `a` does not live long enough
   --> $DIR/propagate-approximated-shorter-to-static-comparing-against-free.rs:30:26
    |
-LL |     let cell = Cell::new(&a);
-   |                ----------^^-
-   |                |         |
-   |                |         borrowed value does not live long enough
-   |                argument requires that `a` is borrowed for `'static`
+LL |       let cell = Cell::new(&a);
+   |                            ^^ borrowed value does not live long enough
 ...
-LL | }
-   | - `a` dropped here while still borrowed
+LL | /     foo(cell, |cell_a, cell_x| {
+LL | |         cell_x.set(cell_a.get()); // forces 'a: 'x, implies 'a = 'static -> borrow error
+LL | |     })
+   | |______- argument requires that `a` is borrowed for `'static`
+LL |   }
+   |   - `a` dropped here while still borrowed
```

Comment:
I'm somewhat worried about the output changes here and in the `async` block in `/issue-72312.nll.stderr`. It also seems like we should also special case their output if we could 🤔

Reply:
*This* case actually seems like an improvement, we "just" need to figure out how to point at the inside of the closure :-/

Reply:
It's kind of weird that this does not point to the start of the closure anymore. Personally I don't really think this makes the diagnostic any less helpful (since the point where the drop happen implies that anyway), but I can try to investigate if you find this important.

Reply:
Would you mind to do so as a follow up? In this case in particular it is an improvement, but I'm not so sure about the other ones.